### DISCUSSION TITLE
How to add comments for mysql field？

### DISCUSSION THREAD

**Discussion by Crownico** [prisma/prisma#23716] [Q&A] [answered]
### Question

```
model User {
  id           Int        @id @default(autoincrement()) 
 }
```
What should I do to generate SQL with comments like this?
```sql
CREATE TABLE `user` (
  `id` int NOT NULL AUTO_INCREMENT COMMENT `this is user id`
) ...
```

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment, marked as answer]
Hi @Crownico 👋 

Thank you for raising this question.

Currently, Prisma Client does not support generating SQL with comments directly from the Prisma schema. This feature has been requested in this  [issue](https://github.com/prisma/prisma/issues/14798). Do leave a 👍  and add your use case as well.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏
